### PR TITLE
Improvements for tables and strange lists

### PR DESCRIPTION
**Pull request by gabyx**

### CODE REVIEW

**Comment by PowerKiKi** [comment]
Tests are broken, could you to fix them please ?

**Comment by gabyx** [comment]
Sure!

**Comment by gabyx** [comment]
Fixed.

File: test_output.markdown [hunk 1]
```diff
@@ -6,9 +6,7 @@ This is a [[wiki link]].
 
 ## Code highlighting is preserved
 
-``` sql
```
Comment (PowerKiKi) [inline test_output.markdown]: Losing syntax highlighting of the correct language is a strong regression. Why did that happen ? Can it be restored ?

**Review by PowerKiKi — CHANGES_REQUESTED** [review]

File: test_output.markdown (hunk 1 above)
Comment (gabyx) [reply test_output.markdown]: Thx, will check this!

**Comment by gabyx** [comment]
Only legacy github markdown supports this in pandoc. I changed it. Tests work now.

File: README.md [hunk 2]
```diff
@@ -10,13 +10,14 @@ enough to get you started. Here are some known limitations:
...
 * Tables without proper headers will be rendered with an empty header
+* Multilines tables will in the first pandoc run converted mostly to HTML which we detect and then in a second might end up as proper markdown tables (if it doesn't work it stays a HTML table) by removing the `<br/>` and afterwards introducing them again into the markdown.
 * Some [interlaced formatting of inline code and bold will be messed up](https://github.com/jgm/pandoc/issues/3024)
 * `<!-- -->` may appear in final ouput in a few places because Redmine incorrectly [does not support HTML in markdown](http://www.redmine.org/issues/20497)
 
 ## Usage
 
 1. Backup your database
-2. [Install pandoc](http://pandoc.org/installing.html)
+2. [Install pandoc](http://pandoc.org/installing.html). Look at the `setup.sh` to use on the server.
```
Comment (PowerKiKi) [inline README.md]:
```suggestion
2. [Install pandoc](http://pandoc.org/installing.html)
```

File: setup.sh [hunk 3]
```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```
Comment (PowerKiKi) [inline setup.sh:1]: This whole file is out of scope. We don't want to maintain instructions for a specific system. It must be deleted entirely.

**Review by PowerKiKi — CHANGES_REQUESTED** [review]

File: setup.sh (hunk 3 above)
Comment (gabyx) [reply setup.sh:1]: Its an example file. Thats all may be just denote it like that.

File: README.md [hunk 4]
```diff
@@ -10,13 +10,15 @@ enough to get you started. Here are some known limitations:
...
+* Multilines tables will in the first pandoc run converted mostly to HTML which we detect and then in a second might end up as proper markdown tables (if it doesn't work it stays a HTML table) by removing the `<br/>` and afterwards introducing them again into the markdown.
 * Some [interlaced formatting of inline code and bold will be messed up](https://github.com/jgm/pandoc/issues/3024)
 * `<!-- -->` may appear in final ouput in a few places because Redmine incorrectly [does not support HTML in markdown](http://www.redmine.org/issues/20497)
 
 ## Usage
 
 1. Backup your database
-2. [Install pandoc](http://pandoc.org/installing.html)
+2. [Install pandoc](http://pandoc.org/installing.html).
+   See the example script `setup.sh` and adapt it to your environment for preparing the conversion.
```
Comment (gabyx) [inline README.md:21]: Denoted as example script. There is also the important regex replace to allow to use HTML inside markdown ...

**Comment by gabyx** [comment]
Can you merge this? I think these are viable improvements for other users...

**Comment by gabyx** [comment]
Fixed the tests.

**Comment by PowerKiKi** [comment]
Thank you for your work on this. I merged it.

Meanwhile I realized I won't be able to keep maintaining this project in the future (as you saw how long it took for me to merge). So I'll archive the entire project. If you have more improvements to do, I suggest you to fork.
